src/timer/CountDownTimer.jsx

```diff
@@ -9,15 +9,44 @@ import { TimerContext } from './TimerProvider';
  */
 const CountDownTimer = injectIntl((props) => {
   const timer = useContext(TimerContext);
+  const timeString = timer.getTimeString();
   const [isShowTimer, showTimer, hideTimer] = useToggle(true);
   const { intl } = props;
 
+  const generateAccessbilityString = (timeState) => {
+    const { hours } = timeState;
```

Comment:
```suggestion
    const { hours, minutes } = timeState;
```
tip: you can do this in one line if you want